plugins/rtmp-services/rtmp-common.c

```diff
@@ -850,6 +853,49 @@ static void rtmp_common_get_max_bitrate(void *data, int *video_bitrate,
 	json_decref(root);
 }
 
+const char **rtmp_common_get_supported_video_codecs(void *data)
```

Comment:
I think this function should be static.
```suggestion
static const char **rtmp_common_get_supported_video_codecs(void *data)
```